estimation/estimate_wct.py

```diff
@@ -390,6 +390,37 @@ def estimate_BB_chours(
     return core_hours, core_hours / data[:, -1]
 
 
+def est_VM_PERT_chours_single(nx: int, ny: int, nz: int, n_cores: int):
+    data = np.array([int(nx) * int(ny) * int(nz), int(n_cores)]).reshape(1, 2)
+
+    core_hours, run_time = est_VM_PERT_chours(data)
+    return core_hours[0], run_time[0]
+
+
+def est_VM_PERT_chours(data: np.ndarray):
+
+    if data.shape[1] != 2:
+        raise Exception(
+            "Invalid input data, has to 3 columns. " "One for each feature."
```

Comment:
"has to 3"?

Reply:
fixed

Reply:
Since this is a copy pasted string - I think we should make it a constant.

Reply:
Maybe use .format as we have different n_columns.

Reply:
good idea. done.